test/CMakeLists.txt

```
@@ -46,6 +55,10 @@ foreach (algo_index RANGE 29)# 43
     if (algo_index STREQUAL 10)
       set (enable_ FALSE)
     endif ()
+    if (algo_index STREQUAL 36 OR algo_index STREQUAL 37
+        OR algo_index STREQUAL 38 OR algo_index STREQUAL 39)
+      set (enable_ FALSE)
+    endif ()
     if (enable_)
       add_test (NAME testopt_algo${algo_index}_obj${obj_index} COMMAND testopt -r 0 -a ${algo_index} -o ${obj_index})
```

```suggestion
      add_test (NAME testopt_algo${algo_index}_obj${obj_index} COMMAND testopt -r 0 -m 1e-3 -a ${algo_index} -o ${obj_index})
```
see if we can fix `ROUNDOFF_LIMITED` errors by adding a tolerance here (rather than just running for 1000 iterations).